src/routes/docs/tooling/command-line/non-interactive/+page.markdoc

```diff
@@ -3,21 +3,36 @@ layout: article
 title: Non-interactive
 description: Deploy changes to Appwrite projects to migrate databases and collections schema, functions, teams, buckets, and more.
 ---
-The Appwrite CLI can be used in a non-interactive and headless manner, without saving configuration or sessions. This is especially useful when you want to automate tasks on a continuous integration server. You can enable the non-interactive mode for the Appwrite CLI by setting the `project ID`, `endpoint`, and `API Key`:
+
+The [Appwrite CLI](/docs/tooling/command-line/installation) can be used in a non-interactive and headless manner, without saving configuration or sessions. This is especially useful when you want to automate tasks on a continuous integration server.
```

Comment:
```suggestion
The [Appwrite CLI](/docs/tooling/command-line/installation) can be used in a non-interactive and headless manner. This is especially useful when you want to automate tasks on a continuous integration server.
```

It still saves to some hidden files. non-interactive here means that you can specify all details with parameters and don't need to use keyboard input during the command.

Also not 100% confident regarding "headless". I think it's true as per definition, just not sure if it's typical for CLI tool to say it's headless. Isnt all CLI tools headless?  🤔